css/templates/travel/elements/_typography.css

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2016 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.travel-spacing-medium {
+  letter-spacing: 0.1em;
+}
+
+.travel-spacing-none {
+  letter-spacing: normal;
+}
+/**
+ * Overide ampstart-base heading line-heights
+ */
+h1.line-height-2, h2.line-height-2, h3.line-height-2, h4.line-height-2, h5.line-height-2, h6.line-height-2,
```

Comment:
i thought we fixed this in some other review

Reply:
done